<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnalysisDialog</class>
 <widget class="QDialog" name="AnalysisDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>340</y>
     <width>181</width>
     <height>29</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="scopeGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>321</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Scope</string>
   </property>
   <widget class="QRadioButton" name="allRadioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>261</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>From first position</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="currentPosRadioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>261</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>From current position</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="timeGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>169</y>
     <width>321</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>How long</string>
   </property>
   <widget class="QCheckBox" name="timeCheckBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Time per move</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="nodesCheckBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Nodes (m)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="depthCheckBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Depth</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>H:mm:ss</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="nodesSpinBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="depthSpinBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>291</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Do you analyze the game record?</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
